tests/integration/DeserializerFactoryTest.php

```diff
@@ -21,9 +23,25 @@ protected function setUp() {
 		$this->factory = TestFactoryBuilder::newDeserializerFactory( $this );
 	}
 
-	public function testGetEntityDeserializerReturnsDeserializer() {
+	public function testNewEntityDeserializerReturnsDeserializer() {
 		$deserializer = $this->factory->newEntityDeserializer();
 		$this->assertInstanceOf( 'Deserializers\Deserializer', $deserializer );
 	}
 
+	public function testNewStatementDeserializerReturnsDeserializer() {
+		$deserializer = $this->factory->newStatementDeserializer();
+		$this->assertInstanceOf( 'Deserializers\Deserializer', $deserializer );
```

Comment:
I like the new PHP 5.5 feature but we should do this once for all tests to stay consistent.